.github/workflows/deploy-non-prod-web.yml

```
@@ -10,16 +10,28 @@ on:
         default: 'QA'
         type: choice
         options:
-          - QA
+          - USQA
           - EUQA
           - USPROD
+          - EUPROD
+      tag:
+        description: "Branch name to deploy (examples: 'main', 'feature/sm')"
+        type: string
+        default: main
```

```suggestion
      tag:
        description: "Branch or Tag name to deploy (examples: 'main', 'feature/sm', 'web-v2023.12.0')"
        type: string
        default: main
```
🎨 Small improvement to the description to help be a bit clearer to the user.